driver/pace/driver/diagnostics.py

```diff
@@ -34,6 +36,35 @@ def cleanup(self):
         ...
 
 
+@dataclasses.dataclass
+class ZSelect:
+    level: int
+    names: List[str]
+
+    def slice_data(self, state: DycoreState):
```

Comment:
technically we select and don't slice here - not super relevant as selecting is a 1-sized slice, but if we actually wan to implement both we might want to rename it here already?

Reply:
Changed it to select data.